Make the `126.diversions' test work when run in the Nix daemon.
See http://thread.gmane.org/gmane.comp.gnu.m4.bugs/2552/focus=2555 for details.

index 02e0f6d..3cd6951 100644
--- a/doc/m4.texinfo
+++ b/doc/m4.texinfo
@@ -4714,7 +4714,7 @@ ifdef(`__unix__', ,
 ')m4exit(`77')')dnl
 changequote(`[', `]')dnl
 syscmd([echo 'divert(1)hi
-format(%1000000d, 1)' | m4 | sed 1q])dnl
+format(%1000000d, 1)' | ]__program__[ | sed -n 1p])dnl
 @result{}hi
 sysval
 @result{}0
